Dockerfile

```diff
@@ -13,7 +13,8 @@ MAINTAINER joseph.schaeffer@autodesk.com
 
 RUN echo "@community http://dl-4.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories && \
     apk add --update python python-dev gfortran py-pip build-base py-numpy@community && \
-    apk del --purge python-dev gfortran py-pip build-base gcc g++ libgcc && \
+    apk del --purge python-dev gfortran build-base gcc g++ libgcc && \
```

Comment:
This should possibly be done in reverse order, so we can still remove py-pip after using it, to leave a smaller image. The question is whether it's still required in order for pytest to still function when called by the Dockerfile - my guess would be that it still has the packages installed even if py-pip gets removed after the install, but it might be good to check/verify. It may also be a trivial size difference, in which case not much to worry about - could you check on those two parts?